### ISSUE TITLE
Image & PDF message Fails: Error 415

### ISSUE THREAD

**Issue by SixPathsLT** [Auties00/Cobalt#513]
Message fails when attempting to send an Image or a PDF file. This issue started happening from version 0.0.6

```
ImageMessage image = new ImageMessageSimpleBuilder().media(attachmentBytes).caption(message).build();
api.sendMessage(contact, image);
```

```
Aug 31, 2024 2:32:11 A.M. it.auties.whatsapp.api.ErrorHandler lambda$defaultErrorHandler$3
SEVERE: [971558117219] Socket failure at MESSAGE
java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: Unexpected response status code: 415
        at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
        at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
        at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1159)
        at java.base/java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:483)
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
Caused by: java.lang.IllegalArgumentException: Unexpected response status code: 415
        at it.auties.whatsapp.net.HttpClient.handlePartialResponse(HttpClient.java:168)
        at it.auties.whatsapp.net.HttpClient.lambda$readPartialResponse$7(HttpClient.java:152)
        at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1150)
        ... 6 more
Aug 31, 2024 2:32:11 A.M. it.auties.whatsapp.api.ErrorHandler lambda$defaultErrorHandler$3
WARNING: [971558117219] Ignored failure
```

**Comment by Auties00** [comment]
This is a bug in the http client, I’ll have it fixed in 0.0.7. I’ll probably release it by the end of the week

**Comment by Auties00** [comment]
Fixed in 0.0.7
